src/storage/page_buffer.c

```diff
@@ -7562,6 +7614,7 @@ pgbuf_claim_bcb_for_fix (THREAD_ENTRY * thread_p, const VPID * vpid, PAGE_FETCH_
     {
       /* Record number of reads in statistics */
       perfmon_inc_stat (thread_p, PSTAT_PB_NUM_IOREADS);
+      pgbuf_Pool.show_status.now.num_pages_read++;
```

Comment:
It also seems that a race condition can occur. In this function, the only latch here is for the hash anchor, but it doesn't prevent it. Similar problems seem to exist for other updating status code.